pom.xml

```diff
@@ -701,7 +701,7 @@
           <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>27.0.1-jre</version>
```

Comment:
This change should be taken separately.

Reply:
This is now merged to the develop branch by Vipin